src/java.desktop/share/classes/javax/swing/plaf/metal/MetalToolTipUI.java

```diff
@@ -61,7 +61,6 @@ public class MetalToolTipUI extends BasicToolTipUI {
      * The space between strings.
      */
     public static final int padSpaceBetweenStrings = 12;
```

Comment:
I guess this can be removed from here and BasicToolTipUI as it is being defined in ToolTipUI